### ISSUE TITLE
check_ping version 2.2.1 Plugin Timeout

### ISSUE THREAD

**Issue by jgbuenaventura** [nagios-plugins/nagios-plugins#419]
Hi,

we recently upgraded our nagios-plugins from version 1.5 to version 2.2.1 and we noticed some difference on check_ping plugin when host is down

on version 1.5
check_ping -H 'ipaddress' -w 3000.0,80% -c 5000.0,100% -p 5 -t 10

PING CRITICAL - Packet loss = 100%|rta=5000.000000ms;3000.000000;5000.000000;0.000000 pl=100%;80;100;0

on version 2.2,1
check_ping -H 'ipaddress' -w 3000.0,80% -c 5000.0,100% -p 5 -t 10

CRITICAL - Plugin timed out

Any comments on this issue?

Thanks!

**Comment by bardahlm** [comment]
I reported the issue a year ago, see #323.

**Comment by orvhub** [comment]
Hi,
As a workaroud I changed the check-host-alive command to
$USER$/check_ping -H $HOSTADDRESS$ -w 1000.10,80% -c 2000.10,100% **-p 1**
the default one was "... -p 5 -t 25"

**Comment by sawolf** [comment]
This issue should be resolved in `master`. If any of you still have this issue after re-compiling, please let me know and I'll re-open this issue.
